pkg/collectors/pool.go

```diff
@@ -127,6 +136,131 @@ func (f *PerfCollector) initPoolDescs() {
 	}
 }
 
+func isPoolInternalMDisks(poolName string, disksList rest.MDisksList) (bool, error) {
+	var internalPool bool
+	for _, disk := range disksList {
+		if poolName == disk[MdiskGroupNameKey].(string) {
+			if disk[ControllerNameKey].(string) != "" {
+				internalPool = false
+			} else {
+				internalPool = true
+			}
+		}
+	}
+	return internalPool, nil
+}
+
+func isCompressionEnabled(poolName string, disksList rest.MDisksList, fsRestClient *rest.FSRestClient) (bool, error) {
+	var compressionEnabled bool
+	disksInPool := mapMDisksToPool(poolName, disksList)
+
+	for _, diskID := range disksInPool {
+		MDiskInfo, err := fsRestClient.LsSingleMDisk(diskID)
+		if err != nil {
+			log.Errorf("get single mdisk info error: %v", err)
+			return compressionEnabled, err
+		}
+		if MDiskInfo[MdiskEffectiveUsedCapacity].(string) == "" {
+			compressionEnabled = false
+		} else {
+			compressionEnabled = true
+		}
```

Comment:
same as [here](https://github.com/IBM/ibm-storage-odf-block-driver/pull/60/files#r1058994256) - is pool considered as compressed according to all disks? only one disk? the last disk?